src/CMakeLists.txt

```diff
@@ -56,6 +56,10 @@ find_package(Threads REQUIRED)
 set(Boost_USE_MULTITHREADED ON)
 
 add_library(_espressopp ${ESPRESSO_SOURCES})
+#target_link_libraries( _espressopp PRIVATE hdf5::hdf5 hdf5::hdf5_hl ) requires CMake 3.19 :(
```

Comment:
We should consider bumping to 3.19 in the CMake refactor.

Reply:
In principle I agree. However, currently, CMake 3.19 must be installed explicitly as it is quite new.

Reply:
I will ship a custom `FindHDF5` until we enforce cmake > 3.19